docs/OIDC/README.md

```diff
@@ -0,0 +1,135 @@
+# OpenID Connect (OIDC) Implementation for uPortal
+
+## Introduction
+OpenID Connect (OIDC) is an authentication layer built on top of OAuth 2.0, enabling clients to verify user identities and obtain basic profile information. This documentation outlines the OIDC possibilities, including available endpoints, group and claims filters, and using OAuth endpoints for clients.
```

Comment:
I don't think this needs an introduction, we can link out to the official OIDC docs or wikipedia for this.
Also uPortal specifically uses form of three legged auth, which doesn't seem to be referenced anywhere